cpp/oneapi/dal/algo/decision_forest/backend/cpu/train_kernel_regression_dense.cpp

```diff
@@ -0,0 +1,182 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <daal/include/services/error_handling.h>
+#include <daal/src/algorithms/dtrees/forest/regression/df_regression_model_impl.h>
+#include <daal/src/services/service_algo_utils.h>
+
+#include <daal/include/algorithms/decision_forest/decision_forest_regression_training_batch.h>
+#include <daal/include/algorithms/decision_forest/decision_forest_regression_training_types.h>
+
+#include <daal/src/algorithms/dtrees/forest/regression/df_regression_train_kernel.h>
+//to prevent reordering by clang-format
+#include <daal/src/algorithms/dtrees/forest/regression/df_regression_train_dense_default_kernel.h>
+
+#include "oneapi/dal/algo/decision_forest/backend/cpu/train_kernel.hpp"
+#include "oneapi/dal/algo/decision_forest/backend/interop_helpers.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+#include "oneapi/dal/detail/common.hpp"
+
+namespace oneapi::dal::decision_forest::backend {
+
+using dal::backend::context_cpu;
+
+namespace df  = daal::algorithms::decision_forest;
+namespace rgr = daal::algorithms::decision_forest::regression;
```

Comment:
Abbreviation - as in examples.

```suggestion
namespace reg = daal::algorithms::decision_forest::regression;
```

Reply:
applied